Manifests/ManagedPreferencesApplications/com.github.macadmins.Nudge.plist

```diff
@@ -157,18 +158,22 @@
 					<string>randomDelay</string>
 					<string>userExperience</string>
 				</array>
+				<key>JSON Preferences File</key>
+				<array>
+					<string>json-url</string>
+				</array>
 			</dict>
 			<key>pfm_type</key>
 			<string>string</string>
 		</dict>
 		<!-- Optional Features -->
```

Comment:
Per our discussion around #567 let's just remove this

Reply:
I've reverted any changes to the Nudge manifest in this PR since a Nudge-dedicated PR #587 now exists and includes changes originally made here along others.